naff/models/discord/enums.py

```diff
@@ -77,7 +77,21 @@ def __iter__(self) -> Iterator:
         yield from _decompose(self.__class__, self)[0]
 
 
-class WebSocketOPCodes(IntEnum):
+SELF = TypeVar("SELF")
+
+
+class CursedIntEnum(IntEnum):
+    @classmethod
+    def _missing_(cls, value) -> SELF:
```

Comment:
This typing with `SELF` is not correct.
```suggestion
    def _missing_(cls: Type[SELF], value) -> SELF:
```